### DISCUSSION TITLE
Seeing Error: b'You need to install postgresql-server-dev-X.Y for building a server-side extension or libpq-dev for building a client-side application.\n'

### DISCUSSION THREAD

**Discussion by abhiramr** [recap-build/recap#74] [Q&A] [answered]
I was trying to install recap-core and I got this error - 
![image](https://user-images.githubusercontent.com/342533/211091620-13f57f3c-2cca-4710-8daa-cd56facb9124.png)

Took me a couple of reads before I realized what I was missing. 

Added the minor guideline in PR #72

**Comment by criccomini** [comment]
Thanks for the PR! This is helpful, though I don't want to put it in the README--there are a lot of these types of issues with Python. Instead, I think I'm going to convert this issue to a Q&A discussion, and put your answer in it. That way it's discoverable from Google.

**Comment by criccomini** [comment, marked as answer]
Per- the PR https://github.com/recap-cloud/recap/pull/72/:

If you get an error saying:

```
site-packages/setuptools/config/setupcfg.py:508: SetuptoolsDeprecationWarning: The license_f      writing manifest file '/tmp/pip-pip-egg-info-d7upprw6/psycopg2.egg-info/SOURCES.txt'
      Error: b'You need to install postgresql-server-dev-X.Y for building a server-side extension or libpq-dev for building a client-side application.\n'
      [end of output]
 ```

You can install libpq-dev using `sudo apt install libpq-dev` and retry installing recap-core.

**Comment by criccomini** [comment]
BTW, there's some more information on this here:

https://www.psycopg.org/docs/install.html

`psycopg2` is currently bundled with Recap to make DB crawling easier, but I think I'm going to eliminate the dependency since we have DuckDB out of the box  now. I've opened https://github.com/recap-cloud/recap/issues/75.
